spectral_fitting/parallel_spectral_fitting.ipynb

```diff
@@ -0,0 +1,1170 @@
+{
+ "metadata": {
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.6.9-final"
+  },
+  "orig_nbformat": 2,
+  "kernelspec": {
+   "name": "Python 3.6.9 64-bit ('casa_py36': conda)",
+   "display_name": "Python 3.6.9 64-bit ('casa_py36': conda)",
+   "metadata": {
+    "interpreter": {
+     "hash": "2d716b7c92d5d4166e3213fb09c67a7f0504f606473c52b1c1916f6fd984541f"
+    }
+   }
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2,
+ "cells": [
+  {
+   "source": [
+    "# Multi-spectrum Parallel Fitting with spectral-cube and astropy #\n",
+    "\n",
+    "## Authors ##\n",
+    "Eric Koch, Adam Ginsburg, Tom Robitaille\n",
+    "\n",
+    "## Learning Goals ##\n",
+    "\n",
+    "* Enabling dask for spectral fitting\n",
+    "* Example of fitting in parallel using dask array functions\n",
+    "\n",
+    "## Keywords ##\n",
+    "\n",
+    "radio astronomy, spectral-line data cubes, spectral fitting, spatial fitting\n",
+    "\n",
+    "## Summary ##\n",
+    "\n",
+    "This tutorial presents a proof-of-concept demonstration of spectral fitting a whole spectral-line data cube. Using dask, this example splits the cube into subcubes and runs the fitting in parallel on these subcubes. Here, we use [astropy.modeling](https://docs.astropy.org/en/stable/modeling/) to fit a single Gaussian to a small region in M33 mapped in CO(2-1) by the ALMA 7-m (ACA) array (Project ID 2019.1.01182.S).\n",
+    "\n",
+    "## Requires ##\n",
+    "\n",
+    "```\n",
+    "pip install scipy\n",
+    "pip install astropy\n",
+    "pip install spectral-cube\n",
+    "pip install radio-beam\n",
+    "pip install dask\n",
+    "pip install dask_image\n",
+    "pip install aplpy\n",
```

Comment:
Simplify imports, it should be possible to give a minimal number of packages on a single line